functions/zram.bats

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bats 
+
+load helpers
+load zram
+
+check_zram_mounts() {
+  local FILE=/etc/ztab
+  local i=0
+
+  if ! [ -f ${FILE} ]; then
```

Comment:
Remove this part "zram not installed, can not test.", shall be handle by the test itself.

Reply:
ok